### DISCUSSION TITLE
github.com 2-factor authentication setup is not working

### DISCUSSION THREAD

**Discussion by arnoldbird** [community/community#37875] [Other Feature Feedback, Questions, & Ideas]
The website hangs at https://github.com/settings/two_factor_authentication/setup/intro in step 2 when I try to set up SMS verification.  I click "Send authentication code" and I see the spinner but nothing happens. I have tried a couple times.  

The browser is Firefox v105.

**Comment by ossanna16** [comment]
Hi there @arnoldbird and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by TroyRobinson** [comment]
Not an answer, although just want to echo the need for a fix. I encountered this same hang on the 2-factor SMS-sending-loading indicator. I tested on Chrome and Edge and same error. I am a Github-approved educator trying to get 2-factor set up so that I can download course materials for my class through Makecode ( https://aka.ms/TEALSintroCS ). These materials require 2-factor set up for some reason.

**Comment by TroyRobinson** [comment]
I tested on a different Windows PC. **Appears to be working now**, whether in-general the server issue was resolved or only in that device, I do not know. In any case, thank you, Github team.
